bindings/wasm/examples/src/0_basic/5_create_vc.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Bip39 } from "@iota/crypto.js";
+import { Client, MnemonicSecretManager } from "@iota/iota-client-wasm/node";
+import { Credential, CredentialValidationOptions, CredentialValidator, FailFast, ProofOptions } from "../../../node";
+import { API_ENDPOINT, createDid } from "../util";
+
+/**
+ * This example shows how to create a Verifiable Credential and validate it.
+ * In this example, alice takes the role of the subject, while we also have an issuer.
+ * The issuer signs a UniversityDegreeCredential type verifiable credential with Alice's name and DID.
+ * This Verifiable Credential can be verified by anyone, allowing Alice to take control of it and share it with whomever they please.
+ */
```

Comment:
These descriptions in the vc and vp example are missing in the Rust example, can you add those? :pray: